app/assets/javascripts/controllers/cloud_object_store_container/cloud_object_store_container_form_controller.js

```diff
@@ -0,0 +1,74 @@
+ManageIQ.angular.app.controller('cloudObjectStoreContainerFormController', ['miqService', 'API', 'storageManagerId', function(miqService, API, storageManagerId) {
+  var vm = this;
+
+  var init = function() {
+    vm.afterGet = false;
+    vm.cloudContainerModel = {
+      name: '',
+      emstype: '',
+      parent_emstype: '',
+      provider_region: '',
+    };
+
+    // fetch StorageManager from querystring
+    if (! isNaN(parseInt(storageManagerId, 10))) {
```

Comment:
@miha-plesko It looks like we can eliminate the `parseInt` related code.
Please see https://github.com/ManageIQ/manageiq-ui-classic/pull/950#issuecomment-294981970 for more details.